gen.treesitter/src/main/java/com/github/gumtreediff/gen/treesitter/AbstractTreeSitterGenerator.java

```diff
@@ -25,12 +25,25 @@
 import com.github.gumtreediff.tree.Tree;
 import com.github.gumtreediff.tree.TreeContext;
 
+import java.io.File;
 import java.io.IOException;
 import java.io.Reader;
 
 public abstract class AbstractTreeSitterGenerator extends ExternalProcessTreeGenerator {
+    // On windows, python commands are usually used to execute the python interpreter
+    // if python is configured in the PATH environment variable
+    public static final String PYTHON = "python";
+    /* In Linux or macOS, Python2 is usually built in for compatibility reasons,
+     * and Python commands are also occupied by Python2 (although Python2 has been removed
+     * from newer distributions), so Python3 commands are usually used to call Python3 interpreters,
+     * although this is not absolute
+     */
+    public static final String PYTHON3 = "python3";
+
     private static final String TREESITTER_CMD = System.getProperty("gt.ts.path",
             "tree-sitter-parser.py");
+    private static final String CUSTOM_PYTHON_PATH = System.getProperty("gt.ts.py.path");
+    private static final String SYSTEM_TYPE = System.getProperty("os.name").toLowerCase();
 
```

Comment:
Hi! I think we could simply all this like this no?
`String PYTHON_CMD = System.getProperty("gt.py.path", "python")`